manifest_lambda/iiifImage.py

```diff
@@ -21,7 +21,7 @@ def thumbnail(self, width="250", height="") -> dict:
 
     def is_image(self):
         mime_type = self.standard_json.get('mimeType', '')
-        if 'pdf' in mime_type or 'audio' in mime_type or 'video' in mime_type:
+        if mime_type is None or 'pdf' in mime_type or 'audio' in mime_type or 'video' in mime_type:
```

Comment:
I think this could be simplified as `if ['pdf', 'audio', 'video', None].includes(mime_type):`, which is also more readable in my opinion